README.md

```diff
@@ -1,3 +1,56 @@
+> 🙌 From now on you can use Algolia Recommend from InstantSearch, simplifying your integration between the two libraries!
+>
+> Learn how to migrate following the [upgrade guide](https://www.algolia.com/doc/guides/algolia-recommend/upgrade-guide).
```

Comment:
Upgrade guides are the following:
- `/doc/guides/algolia-recommend/upgrade/recommend-js/`
- `/doc/guides/algolia-recommend/upgrade/recommend-react/`